tabs/adjustments.js

```diff
@@ -57,14 +57,25 @@ TABS.adjustments.initialize = function (callback) {
         //
         // update selected function
         //
-        
+
         var functionList = $(newAdjustment).find('.functionSelection .function');
-        functionList.val(adjustmentRange.adjustmentFunction);
+        // update list of selected functions
+        var functionListOptions = $(functionList).find('option');
+        var availableFunctionCount = 21; // Available in betaflight 2.9
         
+        if (semver.gte(CONFIG.flightControllerVersion, '3.1.0')) {
+            availableFunctionCount += 1; // RC rate Yaw added to 3.1.0
+        }
+
+        var functionListOptions = $(functionListOptions).slice(0,availableFunctionCount);
```

Comment:
Well, this is actually how it was previously. I didn't rewrite this code, just copied part of the code back that was removed in [PR 193](https://github.com/betaflight/betaflight-configurator/pull/193/files). 
I will however take a closer look at this and see if it can be improved.